utils/types.go

```diff
@@ -24,6 +24,24 @@ type Kubernetes struct {
 	NodeName   string `json:"node_name"`
 }
 
+type Args struct {
+	Mesos Mesos `json:"org.apache.mesos,omitempty"`
+}
+
+type Mesos struct {
+	NetworkInfo NetworkInfo `json:"network_info"`
+}
+
+type NetworkInfo struct {
+	Name string `json:"name"`
+	Labels struct {
```

Comment:
Aye.  My point was solely about readability, which is just as important as re-usability :)